manifests/init.pp

```diff
@@ -162,6 +165,16 @@
     $iftemplate = template('network/ifcfg-eth.erb')
   }
 
+  if $flush {
+    exec { 'network-flush':
+      user        => 'root',
+      command     => "/usr/sbin/ip addr flush dev ${interface}",
```

Comment:
@stepanstipl Why hardcode the path to `ip`?  If someone sets `$flush => true` on RHEL6, Exec['network-flush'] will fail since `ip` is at `/sbin/ip`.